src/server/Document/Renderers/LiteRenderer.tsx

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable react/no-danger */
 import React, { ReactElement, PropsWithChildren } from 'react';
+import trackingScript from '#src/server/utilities/liteATIClickTracking';
```

Comment:
Can we rename this to match the filename?
```suggestion
import liteATIClickTracking from '#src/server/utilities/liteATIClickTracking';
```